### ISSUE TITLE
Itheberg feels maybe misguided?

### ISSUE THREAD

**Issue by tonyaajjackson** [DefenderOfBasic/in-good-faith-handbook#5]
https://github.com/DefenderOfBasic/in-good-faith-handbook/blob/4f97e42c688ec4579fb23e99d90df2cff4cbca40/Book/1%20-%20You/Itheberg%20-%20think%20the%20unthinkable.md

This breaks down in contexts where it's not just obvious that the other person is wrong, but you can see pretty clearly where they made the error that they made in constructing their belief tree.

But also, letting yourself get convinced of your own correctness can blind you to misunderstandings in what you're even talking about, or to errors in your own thoughts.

I've gotten the advice of roughly "performatively pretend that you're more uncertain in your beliefs than you actually are because it makes people more receptive" but I think that's actually bad advice? See this thread:

https://x.com/eshear/status/1773791307552678071?t=C05Po45Y0ydApy_C8kNMGQ&s=19

Maybe there are two messages here?
- Holding some uncertainty about your own correctness can help open you to noticing if you are making an error
- Get curious about why they believe what they believe, rather than performatively acting curious about your own strongly held beliefs about the problem

**Comment by DefenderOfBasic** [comment]
> "performatively pretend that you're more uncertain in your beliefs than you actually are because it makes people more receptive"

yes I do this all the time & **I think it can/does backfire**. Hmmm, I think it DID work a lot for me, *because* I genuinely didn't understand the topic. I only knew what I was told, and I wasn't very confident in it. Once the other side spotted this (because it was true), they started explaining it to more earnestly, exposing their sequence of thought/belief, and I just kept asking them questions, and it did both (1) soften them up to the opposing side (2) help me understand them

**Comment by DefenderOfBasic** [comment]
ok how about this: let's reframe it in a way that is true?

- you are 100% certain this person is wrong, you can *see* the mistake they're making etc
- you do NOT know what you can say to them that will change their mind. You don't know them well enough

I think my entry point into this is to (1) articulate their position, tell a story, see if they confirm it (2) that helps me understand them/helps signal to them that I understand them. not at a superficial level, but at a deeper level

I think this is what I did here in this ["palestinian babies are not innocent"](https://github.com/DefenderOfBasic/works-in-progress/discussions/2) interaction, the part where I said the words I believed he was thinking (kill them all, there's no other solution). This helped me (1) confirm my understanding (2) prove to him that I understand him, we built a rapport on that 

<img src="https://github.com/user-attachments/assets/293ef0dd-0769-4158-bb76-c329b58d4948" width="450" />
